### PR TITLE
Setup new arm

### PR DESCRIPTION
**Pull request by SyrianSpock**
This PR removes all the code hacked last year to handle the additional degrees of freedom that were needed. This year we currently have a scara arm and a pump as end effector.

Tested on hardware and working, the arm PIDs need tuning though.

### CODE REVIEW

File: master-firmware/src/arms/arms_controller.c [hunk 1]
```diff
@@ -17,10 +16,6 @@
 
 
 scara_t left_arm;
```
Comment (antoinealb) [inline master-firmware/src/arms/arms_controller.c]: maybe we should rename `left_arm` to like `main_arm` or maybe just `arm`?

File: master-firmware/src/arms/arms_controller.c [hunk 2]
```diff
@@ -41,64 +36,20 @@ void arms_init(void)
     static cvra_arm_motor_t left_z = {.id = "left-z", .direction = 1, .index = 0};
```
Comment (antoinealb) [inline master-firmware/src/arms/arms_controller.c]: same here, we should rename `left_*` to `main_*`

File: master-firmware/src/main.c [hunk 3]
```diff
@@ -308,24 +306,7 @@ void init_arm_motors(void)
...
-    motor_manager_create_driver(&motor_manager, "left-wrist-down");
-    motor_manager_create_driver(&motor_manager, "right-wrist-up");
-    motor_manager_create_driver(&motor_manager, "right-wrist-down");
-
-    motor_manager_create_driver(&motor_manager, "right-element-rotate");
-    motor_manager_create_driver(&motor_manager, "left-element-rotate");
-
-    bus_enumerator_add_node(&bus_enumerator, "left-hand", NULL);
-    bus_enumerator_add_node(&bus_enumerator, "right-hand", NULL);
+    motor_manager_create_driver(&motor_manager, "left-pump-5");
```
Comment (antoinealb) [inline master-firmware/src/main.c]: The board names should be changed in the bootloader config.

File: master-firmware/src/arms/arms_controller.c (hunk 1 above)
Comment (SyrianSpock) [reply master-firmware/src/arms/arms_controller.c]: I guess `arm` is enough.

**Comment by antoinealb** [comment]
Good updates! Wanna test on real robot before merging ?

**Comment by SyrianSpock** [comment]
Yes sure I'll do that on thursday.

**Comment by SyrianSpock** [comment]
Tested and working on hardware. Had to fix some minor things.

**Pull request merged**